server/api.py

```
@@ -0,0 +1,312 @@
+    return {"message": "Badda-Bing Badda-Bang"} 
+
+@app.post("/start")
+async def start_model(request: StartRequest):
+    """Load the model and start the agent"""
+    global _messages, _runner
+    print(str(request))
+    _messages = [ChatMessage(role="system", content=SYSTEM_PROMPT)]
+
+    try:
+        _runner = get_or_load_model(request.model)
+        return {"message": "Model loaded"}
```

Ruff (0.14.1)

Consider moving this statement to an `else` block

(TRY300)